### PR TITLE
Resolves #151 Issue with scatter_ppm

### LINKED ISSUE #151: New scatter_ppm parameter does not work with shared fits
As of right now the new scatter_ppm parameter in Stage 5 introduced in PR #131 does not work properly for shared fits. Only the scatter_ppm parameter gets used for all wavelengths while the scatter_ppm_# parameters are ignored.

### PR DESCRIPTION
**Pull request by cpiaulet**
Make the scatter_ppm parameter robust to having >1 wavelength channel being fitted. Individual scatter_ppm params are created and used for each channel.

### CODE REVIEW

File: eureka/S5_lightcurve_fitting/fitters.py [hunk 1]
```diff
@@ -262,8 +268,11 @@ def emceefitter(lc, model, meta, log, **kwargs):
 
     model.update(fit_params, freenames)
     if "scatter_ppm" in freenames:
-        ind = np.where(freenames == "scatter_ppm")
-        lc.unc_fit = medians[ind[0][0]]*1e-6
+        ind = [i for i in np.arange(len(freenames)) if freenames[i][0:11] == "scatter_ppm"]
+        lc.unc_fit = np.ones_like(lc.flux) * medians[ind[0]] * 1e-6        
+        if len(ind)>1:
+            for chan in np.arange(lc.flux.size//lc.time.size):
+                lc.unc_fit[chan*lc.time.size:(chan+1)*lc.time.size] = medians[ind[chan]] * 1e-6
```
Comment (taylorbell57) [inline eureka/S5_lightcurve_fitting/fitters.py:275]: Does a similar change to this not also need to be made for the other fitters?

**Review by taylorbell57 — CHANGES_REQUESTED** [review]
I haven't run this, but I think changes similar to the ones you made for emcee need to be made for all the other fitters

**Comment by cpiaulet** [comment]
right, I'll get on that :)

**Comment by cpiaulet** [comment]
@taylorbell57 I just put it in and tested for lsq and dynesty. I also put it in for lmfit but it does not seem to work either way.

**Comment by taylorbell57** [comment]
@cpiaulet can you merge the main branch (presumably the remote/main branch for you) into your branch to resolve the merge conflicts and so that I can test out your code?

**Review by taylorbell57 — APPROVED** [review]
Alright, I tried this out in a local branch and it seems to work, so you have my approval. You'll still need to resolve the merge conflicts before you can merge this into main though

**Comment by cpiaulet** [comment]
done! I had only merged it on the date I opened the pull request :)

**Review by taylorbell57 — APPROVED** [review]

**Pull request merged**